.github/workflows/test_suite_mac.yml

```diff
@@ -107,10 +90,10 @@ jobs:
         hdiutil mount artifact/webots-*.dmg
         sudo cp -R /Volumes/Webots/Webots.app /Applications
         hdiutil unmount /Volumes/Webots
-    - name: Set up Python 3.9
+    - name: Set up Python 3.10
       uses: actions/setup-python@v2
       with:
-        python-version: 3.9
+        python-version: 3.10
```

Comment:
Is there a reason to use Python 3.10 over 3.11? Is it not available on macOS CI machines yet?

Reply:
I have the feeling that 3.11 is still very new and a few modules may be missing (not yet ported to 3.11).

Reply:
I just set 3.11. Let's see the tests are passing.

Reply:
I seems it worked out, so let's keep Python 3.11.